pgml-cms/docs/api/sql-extension/pgml.train/README.md

```diff
@@ -63,7 +63,7 @@ This will create a "My Classification Project", copy the `pgml.digits` table int
 
 When used for the first time in a project, `pgml.train()` function requires the `task` parameter, which can be either `regression` or `classification`. The task determines the relevant metrics and analysis performed on the data. All models trained within the project will refer to those metrics and analysis for benchmarking and deployment.
 
-The first time it's called, the function will also require a `relation_name` and `y_column_name`. The two arguments will be used to create the first snapshot of training and test data. By default, 25% of the data (specified by the `test_size` parameter) will be randomly sampled to measure the performance of the model after the `algorithm` has been trained on the 75% of the data.
+The first time it is called, the function will also require a `relation_name` and `y_column_name`. The two arguments will be used to create the first snapshot of training and test data. By default, 25% of the data (specified by the `test_size` parameter) will be randomly sampled to measure the performance of the model after the `algorithm` has been trained on the 75% of the data.
```

Comment:
Why do we prefer not using contractions?

Reply:
I suggested it as a way to make it simpler for non-native English speakers to read.

Another reason is for translation, but I figured you probably have no plans for that at this point.